third_party/move/tools/move-mutator/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+use clap::{Arg, Command};
+use serde::{Deserialize, Serialize};
+
+/// Move mutator options
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct Options {
+    /// The paths to the Move sources.
+    pub move_sources: Vec<String>,
+}
+
+impl Default for Options {
+    fn default() -> Self {
+        Self {
+            move_sources: vec![],
+        }
+    }
+}
+
+impl Options {
+    /// Creates options from toml configuration source.
+    pub fn create_from_toml(toml_source: &str) -> anyhow::Result<Options> {
```

Comment:
nit suggestion:
we could drop the `create_` since `from_xxx` implies we are creating an Option (also the return type indicates it as well)
```suggestion
    pub fn from_toml(toml_source: &str) -> anyhow::Result<Options> {
```

Reply:
Changed.